app/models/payment_transaction.rb

```diff
@@ -34,6 +34,9 @@ def process!(credit_card_auth_code, gateway, gateway_options)
                    when ActiveMerchant::Billing::AuthorizeNetGateway
           logger.info("Purchasing with authorize.net")
           purchase_with_authorize_net(credit_card_auth_code, gateway)
+                   when ActiveMerchant::Billing::StripePaymentIntentsGateway
+          logger.info('Purchasing with stripe (StripePaymentIntentsGateway)')
```

Comment:
How about getting rid of lines 35, 38, 41 and 44 and just do the following right before the `case` clause?

```ruby
logger.info("Purchasing with #{gateway.class}")
```

I understand that would change the other (existing) log messages, on top of which maybe some monitoring might have been set, but I think that's acceptable to clean this up a bit. Besides, it would allow more assertive log parsing if we used the name of the class always, like you are now proposing with "StripePaymentIntentsGateway".

WDYT?

Reply:
I like it 👍🏼 But I prefer to leave this kind of minor improvements for the following PR 👍🏼